doc/website/blog/2023-12-12-hibernate-schema-loader.mdx

```diff
@@ -0,0 +1,591 @@
+
+```sql
+-- Create "movies" table
+-- Create "movies" table
+CREATE TABLE "movies" (
+  "id" bigserial NOT NULL,
+  "numberinseries" integer NULL,
+  "title" character varying(255) NULL,
+  PRIMARY KEY ("id")
+);
+-- Create "actors" table
+CREATE TABLE "actors" (
+  "name" character varying(255) NOT NULL,
+  PRIMARY KEY ("name")
+);
+-- Create "movieparticipation" table
+CREATE TABLE "movieparticipation" (
+  "actorname" character varying(255) NOT NULL,
+  "movieid" bigint NOT NULL,
+  PRIMARY KEY ("actorname", "movieid"),
+  CONSTRAINT "fkaq2kkwvh9870847sm35vtjtiy" FOREIGN KEY ("movieid") REFERENCES "movies" ("id") ON UPDATE NO ACTION ON DELETE NO ACTION,
+  CONSTRAINT "fktm8fbwa577lnbvwdjegwxvget" FOREIGN KEY ("actorname") REFERENCES "actors" ("name") ON UPDATE NO ACTION ON DELETE NO ACTION
+);
+```
+
+</TabItem>
+<TabItem value="migrations/atlas.sum">
+
+```
+h1:Ezh6r7A0pU4XS7PztE87h+aq5PoGGVTd6kaprtpXxas=
+20231211114848.sql h1:9ECs2QsjHE8S4PxXxFcoPYaOVUWRjpxHuF8MGFRd3dE=
+```
+
+</TabItem>
+</Tabs>
+
+Atlas uses the `atlas.sum` file to protect against conflicting schema changes, you can read about it [here](/versioned/new#recalculating-the-directory-hash).
+
+:::note
+Currently, Atlas does not support using generated fields that require data initialization such as `GenerationType.SEQUENCE`, `GenerationType.TABLE`, and `Generation.AUTO`.
+
+If needed, you can still export the schema using the flag --enable-table-generators (or -Denable-table-generators using Maven). When applying the schema to your database, you will need
+to make sure to apply the ignored statements (using `atlas migrate --env hibernate diff --edit`). See more information on manual migrations [here](/versioned/diff)
+
+For example, if you are adding GenerationType.SEQUENCE to the Event entity, you will need to add insert statements to your generated migration file:
+
+```diff
+diff --git a/migrations/20231210140844.sql b/examples/with_local_plugin_repository/migrations/20231210140844.sql
+index ad80a64..5955834 100644
+--- a/migrations/20231210140844.sql
++++ b/migrations/20231210140844.sql
+@@ -4,3 +4,6 @@ CREATE TABLE `Event` (`id` bigint NOT NULL AUTO_INCREMENT, `title` varchar(255)
+ -- Create "Event_SEQ" table
+ CREATE TABLE `Event_SEQ` (`next_val` bigint NULL) CHARSET utf8mb4 COLLATE utf8mb4_0900_ai_ci;
++ -- Initialize "Event_SEQ" table
++ insert into Event_SEQ values ( 1 );
+```
+
+Testing these changes can be done by running the application with a local database and creating the entity. To apply the migration directory
+to the local database, use `atlas migrate apply`.
+:::
+
+### Testing the migrations
+
+Having a migration directory is nice and dandy, but we can also apply it to a database without breaking a sweat.
+Let's start a local PostgreSQL instance:
+
+```bash
+docker run -it --rm --name mypostgres -p 5432:5432 -e 'POSTGRES_PASSWORD=password' postgres
+```
+
+And let the magic happen by applying our schema to the database:
```

Comment:
```suggestion
Next, let's apply our migrations to the database:
```
Technically, we aren't applying the schema (which is done with the `schema apply` command).